tipsy-bot/tipsy.py

```diff
@@ -1,66 +1,73 @@
 import asyncio
 import os
-import time
 
 from viam.robot.client import RobotClient
 from viam.rpc.dial import Credentials, DialOptions
 from viam.components.sensor import Sensor
 from viam.components.base import Base
 from viam.services.vision import VisionClient
 
-robot_secret = os.getenv('ROBOT_SECRET') or ''
-robot_address = os.getenv('ROBOT_ADDRESS') or ''
-#change this if you named your base differently in your robot configuration
-base_name = os.getenv('ROBOT_BASE') or 'tipsy-base'
-#change this if you named your camera differently in your robot configuration
-camera_name = os.getenv('ROBOT_CAMERA') or 'cam'
-pause_interval = os.getenv('PAUSE_INTERVAL') or 3
+robot_secret = os.getenv("ROBOT_SECRET") or ""
+robot_address = os.getenv("ROBOT_ADDRESS") or ""
+# change this if you named your base differently in your robot configuration
+base_name = os.getenv("ROBOT_BASE") or "tipsy-base"
+# change this if you named your camera differently in your robot configuration
+camera_name = os.getenv("ROBOT_CAMERA") or "cam"
+# change this if you named your sensors differently in your robor configuration
+sensor_names = (os.getenv("SENSOR_NAMES") or "ultrasonic,ultrasonic2").split(",")
```

Comment:
It might be more consistent to call this `ROBOT_SENSORS`. 🤔

Reply:
i agree